hashtable.csv

```diff
@@ -10,3 +10,14 @@ code, name
 0x19, shake-256
 0x40, blake2b
 0x41, blake2s
+0x55, md5
+0x60, xxHash32
+0x61, xxHash64
+0x62, Spooky32
+0x63, Spooky64
+0x64, Spooky128
+0x65, Murmur3A
+0x66, Murmur3C
+0x67, Murmur3F
+0x75, crc32
+0x76, crc32-C
```

Comment:
i think we should source more of the "non cryptographic hashes".  List of more:
- http://jpountz.github.io/lz4-java/1.2.0/xxhash-benchmark/
- https://github.com/rurban/smhasher

and
- more crypto hashes: https://bench.cr.yp.to/results-hash.html